### ISSUE TITLE
python ctypes does not find c lib, prevents ansible to start

### ISSUE THREAD

**Issue by rkohser** [vegardit/cygwin-portable-installer#39]
Hi, 
When selecting INSTALL_ANSIBLE=yes and ANSIBLE_GIT_BRANCH=stable-2.14, I systematically get the same error at ansible startup stating that function wcwidth cannot be found.
This comes from this line https://github.com/ansible/ansible/blob/stable-2.14/lib/ansible/utils/display.py#L47 which is failing to find 'c' library.
The test can be easily done like this 
`python  -c 'import ctypes.util; print(ctypes.cdll.LoadLibrary(ctypes.util.find_library("c")))'`
returning currently
`<CDLL 'None', handle 100400000 at 0x6fffffec7210>`

I thought it was a Cygwin issue but a "classic" install of cygwin does not have the issue and returns the cygwin libc implementation like expected
`<CDLL 'cygwin1.dll', handle 180040000 at 0x6ffffffa4ca0>`

I looked at differences in the environment, did a bunch of tests and could not find the right variable to set or package to install.
I guess there is some dependency missing in the config file.

NB: the wcwidth usage has been introduced in ansible 2.11, picking this version solved the issue but I am still looking at how to make 2.14 work.

Any idea how I could solve this ?

**Comment by sebthom** [comment]
Hmm, you can play around with the PATH variable set in `cygwin-portable.cmd` maybe something is missing.

**Comment by sebthom** [comment]
I just did a fresh non-portable install of cygwin on a clean machine and I also get `<CDLL 'None',` when executing the Python script you provided. So I have doubts that issue is related to the portable installation.

![image](https://user-images.githubusercontent.com/426959/218199295-8c98e4c9-2217-444b-9fee-cfc7cacdeace.png)

**Comment by rkohser** [comment]
After playing around with the different packages and diffing some cygcheck reports I found out the missing package is **cygwin-devel**. The wcwidth call makes it a dependency for the 2.11+ versions of ansible.
I guess the package must embed the dynamic linking tools ctypes is based on to locate the libraries. see https://docs.python.org/3/library/ctypes.html#finding-shared-libraries 
Also, I got an error stating that ansible required a 3.9+ version of python.
With cygwin-devel and the python39 packages; ansible is working fine.
